README.md

```diff
@@ -1,30 +1,30 @@
-# OpenTF
+# OpenTofu
 
-- Manifesto: https://opentf.org
-- About the OpenTF fork: https://opentf.org/fork
+- Manifesto: https://OpenTofu.org
+- About the OpenTofu fork: https://opentf.org/fork
 - [Join our Slack community!](https://join.slack.com/t/opentfcommunity/shared_invite/zt-237chyryd-mFULiefrbYGMYQoG72BUpQ)
 
-<img alt="OpenTF" src="https://raw.githubusercontent.com/opentffoundation/brand-artifacts/main/full/transparent/SVG/on-light.svg" width="600px">
+<img alt="OpenTofu" src="https://raw.githubusercontent.com/opentffoundation/brand-artifacts/main/full/transparent/SVG/on-light.svg" width="600px">
```

Comment:
This one will be changed by my issue please do not alter it #457 as names might change